students/arkadiusz_kasprzyk/lesson_06_dicts/guess_the_number.py

```diff
@@ -0,0 +1,32 @@
+import random
+
+
+def guess(N):
+    print("Give your guess (number between 1 " +
+          "and {} or one of '0', '?', 'h', 'help' for help):".format(N))
+    guesset = input("").split(" ")
```

Comment:
This is ok, but you could have stored the prompt in a variable and then call `input` on it instead of a separate print.